Formula/mikutter.rb

```diff
@@ -22,7 +22,7 @@ class Mikutter < Formula
 
   depends_on "gobject-introspection"
   depends_on "gtk+3"
-  depends_on "ruby@2.7"
+  depends_on "ruby@3.0"
```

Comment:
Having an issue with `gobject-introspection` gem for Ruby 3.1/3.2. May try updating dependencies to see if it helps.